File: Subject Archive/INFO1910 Intro to programming/transpiler/test/testcase/while_4.py
import sys
def main(argc, argv):

    asdf = int( 20)

    while(asdf != 10):
        asdf = asdf - 1
        x = int( asdf/4)

        print("x i %d"%(x), end = "")
        print("asdf is %d\n"%( asdf), end = "")
        
    return 0
    
CALLMAINFUNCTIONVAR = main(len(sys.argv), sys.argv)

File: Subject Archive/INFO1910 Intro to programming/transpiler/test/while/while_4.c
#include <stdio.h>

int main()
{
    int asdf = 20;

    while(asdf != 10)
    {
        asdf = asdf - 1;
        int x = asdf/4;
        printf("x i %d",x);
        printf("asdf is %d\n", asdf);
    }
    return 0;
}